go-fuzz-build/cover.go

```diff
@@ -33,6 +30,26 @@ func instrument(pkg, fullName string, fset *token.FileSet, parsedFile *ast.File,
 	}
 	if sonar == nil {
 		file.addImport("go-fuzz-dep", fuzzdepPkg, "CoverTab")
+		for _, element := range info.Types {
+			_, named := element.Type.(*types.Named)
+			if named {
+				importPackage := element.Type.(*types.Named).Obj().Pkg()
+				if importPackage != nil {
+					importPath := importPackage.Path()
+					foundImport := false
+					for _, imp := range file.astFile.Imports {
+						impPath := imp.Path.Value
+						impPath = imp.Path.Value[1 : len(impPath)-1]
+						if impPath == importPath {
+							foundImport = true
+						}
+					}
+					if !foundImport && pkg != importPath && !strings.Contains(importPath, "internal/") {
```

Comment:
What's the `strings.Contains(importPath, "internal/")` for?

Reply:
That part of the if statement can be removed as every unused import will be removed by goimports. Before using the goimports I thought only imports with the internal prefix were added incorrectly.